CMake/add_project_kwiver.cmake

```diff
@@ -73,14 +73,18 @@ ExternalProject_Add(kwiver
     -DKWIVER_ENABLE_DARKNET:BOOL=${VIAME_ENABLE_YOLO}
     -DKWIVER_ENABLE_C_BINDINGS:BOOL=${VIAME_ENABLE_PYTHON}
     -DKWIVER_ENABLE_PYTHON:BOOL=${VIAME_ENABLE_PYTHON}
+    -DKWIVER_SYMLINK_PYTHON:BOOL=${VIAME_SYMLINK_PYTHON}
     -DKWIVER_ENABLE_CUDA:BOOL=${VIAME_ENABLE_CUDA}
     -DKWIVER_ENABLE_DOCS:BOOL=${VIAME_ENABLE_DOCS}
     -DKWIVER_INSTALL_DOCS:BOOL=${VIAME_ENABLE_DOCS}
     -DKWIVER_ENABLE_TRACK_ORACLE:BOOL=${VIAME_ENABLE_KWANT}
+    -D KWIVER_ENABLE_TESTS:BOOL=On
```

Comment:
KWIVER enable tests should not be enforced to always on, we should add a high level viame flag which also controls this